[Beam-1232] Fixed pipeline.py comments to be conformant to python-sdk

yk5 opened the pull request:
Hi @robertwb, I just started to work on apache-beam python sdk b/33761836, and found some examples mismatch to the syntax, so fixing. can you please take a look?

Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [x] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [x] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [x] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

aaltay commented on sdks/python/apache_beam/pipeline.py:
```diff
@@ -28,15 +28,18 @@
 
   # Create a pipeline object using a local runner for execution.
-  pipeline = Pipeline(runner=DirectPipelineRunner())
+  p = beam.Pipeline('DirectPipelineRunner')
```
I believe you need to rebase. DirectPipelineRunner was renamed to DirectRunner.

aaltay commented on sdks/python/apache_beam/pipeline.py:
```diff
@@ -28,15 +28,18 @@
   # Add to the pipeline a "Create" transform. When executed this
   # transform will produce a PCollection object with the specified values.
-  pcoll = pipeline.create('label', [1, 2, 3])
+  c = p | 'label' >> beam.Create([1, 2, 3])
```
You can omit the label or give at a more meaningful label. Either one of the things would be fine:
```
pcoll = p | 'create' >> beam.Create([1, 2, 3])
pcoll = p | beam.Create([1, 2, 3])
```
I think I prefer `pcoll` over `c` as a variable name. We use `p` for pipeline in general but for other things we have more explicit names.

yk5 replied:
Done. Thanks for letting me know!

aaltay commented on sdks/python/apache_beam/pipeline.py:
```diff
@@ -28,15 +28,18 @@
-  pcoll = pipeline.create('label', [1, 2, 3])
+  c = p | 'label' >> beam.Create([1, 2, 3])
+
+  # Write to a file. The filename will be formatted as output-00000-of-00001.
```
Do we need the write part for this little example? It does not add much to how pipeline is used beyond the previous example and needs to deal with file output formats.

yk5 replied:
I added some more comments to explain. I feel like the example in the current shape is incomplete since it does nothing. How about this?

yk5 commented:
Thanks for review, Ahmet!

I addressed comments in my local repo and pushed to the branch by following commands.
```
$ git commit -a
$ git pull --rebase # and then resolved conflicts.
$ git rebase --continue
$ git push yk5 +python-sdk
```
Am I doing right? 
PTAL..

aaltay commented on sdks/python/apache_beam/pipeline.py:
```diff
@@ -28,15 +28,19 @@
-  pcoll = pipeline.create('label', [1, 2, 3])
+  pcoll = p | 'create' >> beam.Create([1, 2, 3])
+
+  # Another transform to pcoll, e.g., writing to a text file.
```
Maybe `Another transform could be applied to pcoll` instead of `Another transform to pcoll`

yk5 replied:
Done. Thanks!

aaltay commented:
@yk5 Yes, that is correct. Thank you for the update.

yk5 commented:
Addressed the comment. Can you PTAL? Thanks!

aaltay commented:
LGTM.

R: @robertwb

robertwb commented:
Thanks. This has been merged, you can now close this PR.

yk5 commented:
Thank you guys! Closing..
